website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -67,17 +67,17 @@ sync to understand how the syncing works.
 The sync process must authenticate to both Kubernetes and Consul to read
 and write services.
 
-For Consul, the process accepts both the standard CLI flag `-token` and
```

Comment:
Does it not read from the CLI arg anymore?

Reply:
Since this refers to the `consul-k8s` process that runs the sync, it currently does not support the CLI flag (seems to have been an implementation oversight). The env variable is not supported in the Helm chart currently either, but there's PR up for adding it, so that will be the recommended path moving. I chatted with Paul and he recommends this approach from a security standpoint.